seqr/management/commands/check_for_new_samples_from_pipeline.py

```diff
@@ -350,6 +359,59 @@ def _update_pdos(session, project_guid, sample_ids):
 
         return sorted(pdos_to_create.keys())
 
+    @classmethod
+    def update_individuals_sample_qc(cls, sample_type, projects, sample_qc_map):
+        sample_individual_map = {
+            i.sample.sample_id: i for i in Individual.objects.filter(
+                family__project__in=projects,
+                sample__sample_id__in=sample_qc_map.keys(),
+            ).select_related('sample')
+        }
+
+        updated_individuals = []
+        unknown_filter_flags = set()
+        unknown_pop_filter_flags = set()
+
+        for sample_id, individual in sample_individual_map.items():
+            record = sample_qc_map[sample_id]
+            filter_flags = {}
+            for flag in json.loads(record['filter_flags']):
+                flag = '{}_{}'.format(flag, sample_type) if flag == 'coverage' else flag
+                flag_col = QC_FILTER_FLAG_COL_MAP.get(flag, flag)
+                if flag_col in record:
+                    filter_flags[flag] = record[flag_col]
+                else:
+                    unknown_filter_flags.add(flag)
+
+            pop_platform_filters = {}
+            for flag in json.loads(record['qc_metrics_filters']):
+                flag_col = 'sample_qc.{}'.format(flag)
+                if flag_col in record:
+                    pop_platform_filters[flag] = record[flag_col]
+                else:
+                    unknown_pop_filter_flags.add(flag)
+
+            individual.filter_flags.update(filter_flags)
+            individual.pop_platform_filters.update(pop_platform_filters)
+            individual.population.update(record['qc_pop'].upper())
+            updated_individuals.append(individual)
+
+        if updated_individuals:
+            Individual.objects.bulk_update(
```

Comment:
Its better to use the `Individual.bulk_update_models` helper as it includes audit logging by default